### PR TITLE
Add configurable activation function to MLP models #41

### PR DESCRIPTION
**Pull request by JARACH-209**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue
Adding configurable activation functions

https://github.com/facebookresearch/mbrl-lib/issues/41#issue-812559086

## How Has This Been Tested (if it applies)
added unit test and ran pytest core module

## Checklist

- [ ] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](../../CONTRIBUTING.md) document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by JARACH-209** [comment]
@luisenp  I have added the unit test and covered both activation functions, with parameters and without.

File: tests/core/test_models.py [hunk 1]
```diff
@@ -19,6 +19,33 @@
 _DEVICE = "cuda:0" if torch.cuda.is_available() else "cpu"
 
 
+def test_activation_functions_guassian_mlp():
+    activation_cfg_silu = omegaconf.OmegaConf.create(
```
Comment (luisenp) [inline tests/core/test_models.py:23]: Looks good! One final comment, just add a test that when `use_silu=True` the given config is ignored. After that I'll run some full training tests and then merge. Thanks!

Comment (JARACH-209) [reply tests/core/test_models.py:23]: @luisenp Done!

**Comment by JARACH-209** [comment]
@luisenp Hi! I hope changes were sufficient .

**Comment by luisenp** [comment]
Hi @JARACH-209! Yes, this looks good. I'll run some test before merging, but the code should be in some time this week. Thanks a lot!

**Pull request merged**